recipes/cpptrace/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir, replace_in_file
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+import os
+
+required_conan_version = ">=1.53.0"
+
+class PackageConan(ConanFile):
+    name = "package"
```

Comment:
Didnt catch it at first: this should be the lib's name (the description field also need filling)

Reply:
Ope thank you? That fixed at least one weird error I was seeing locally. Sorry to bother you with this but when I try building I'm seeing

```-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: Target declared 'cpptrace::cpptrace'
CMake Error at build/gcc-9-x86_64-17-release/generators/cmakedeps_macros.cmake:67 (message):
  Library 'package_lib' not found in package.  If 'package_lib' is a system
  library, declare it with 'cpp_info.system_libs' property
Call Stack (most recent call first):
  build/gcc-9-x86_64-17-release/generators/cpptrace-Target-release.cmake:24 (conan_package_library_targets)
  build/gcc-9-x86_64-17-release/generators/cpptraceTargets.cmake:26 (include)
  build/gcc-9-x86_64-17-release/generators/cpptrace-config.cmake:16 (include)
  CMakeLists.txt:5 (find_package)


-- Configuring incomplete, errors occurred!

*********************************************************
Recipe 'cpptrace/0.2.0 (test package)' cannot build its binary
It is possible that this recipe is not Conan 2.0 ready
If the recipe comes from ConanCenter, report it at https://github.com/conan-io/conan-center-index/issues
If it is your recipe, check if it is updated to 2.0
*********************************************************

ERROR: cpptrace/0.2.0 (test package): Error in build() method, line 21
        cmake.configure()
        ConanException: Error 1 while executing
```

Not quite sure what I'm doing wrong

Reply:
Oh, never mind, silly mistake on my end :)